.github/workflows/ci.yaml

```
@@ -168,9 +170,9 @@ jobs:
           cp .github/ci_resources/test_config_template.json tests/src/test/resources/config/local.json
           sed -i "s:WORK_DIR:$(pwd):" .github/ci_resources/tests-job.yaml
           sudo -E kubectl apply -f .github/ci_resources/tests-job.yaml
-          sudo -E kubectl wait --for=condition=complete --timeout=300s job/ci-tests -n extractor &
+          sudo -E kubectl wait --for=condition=complete --timeout=600s job/ci-tests -n extractor &
```

Do the tests actually take this long now? Ok if so, but if this was added bc of the python worker SDK issue, let's pull it back out to fail fast(er).

---

They don't take quite this long, but 5 minutes is definitely not safe as-is. One of the runs on GHA from this branch took 6 minutes and 9 seconds for the test pod to be done. The primary reason the time jumped up is the test to make sure we don't get duplicated rows. That test has to launch and wait for another full run of the whole ingest job. We could speed that test up a bit by launching on a smaller subset of data, but that would be at the expense of the quality of the checks (if some data caused duplicate rows but not others).

---

Oh, and actually the timeout here was only the relevant one just _barely_ with the python worker SDK issue. There's a wait here for 5 minutes, but also within the tests there's a wait of 5 minutes for the launched ingest job to complete. This one gets triggered first because it takes a few seconds before the one within the tests starts polling. So, even if we had a similar issue, we wouldn't go much above 5 minutes here anyway.